google/auth/transport/_mtls_helper.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helper functions for getting mTLS cert and key, for internal use only."""
```

Comment:
I assume by "for internal use only" you mean for use inside Google Cloud? Because if it was truly internal only, it wouldn't be in this library. So you can either just remove that part of the sentence entirely, or explain better what this is for.

Reply:
I think what this means is that this class should not be used directly by end users (also signaled by the file name starting with an underscore).

Reply:
yes it means this file is not a public interface as indicated by the leading underscore in the file name.

Reply:
Sounds like this is idiomatic to python? If so then my point is wrong.

Reply:
Yes. See also
https://github.com/googleapis/google-auth-library-python/blob/master/google/auth/transport/_http_client.py#L15